wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/AccountViews/AccountViewFactory.swift

```diff
@@ -20,9 +20,9 @@ import WireDataModel
 
 enum AccountViewFactory {
 
-    static func viewFor(account: Account, user: ZMUser? = nil, displayContext: DisplayContext) -> AccountView {
+    static func viewFor(account: Account, user: ZMUser? = nil, displayContext: DisplayContext) -> AccountView! {
```

Comment:
By moving the exclamation mark here here the caller can still decide to force-unwrap or not.